src/openforms/js/components/formio_builder/translation.js

```diff
@@ -56,37 +56,22 @@ export const extractComponentLiterals = component => {
     // Do not show warnings for date/datetime placeholders, they are not translatable:
     .filter(property => (isDateOrDatetimeComponent ? property !== 'placeholder' : property))
     .filter(property => !isRegex(property))
-    .map(property => {
-      return {
-        property: property,
-        literal: get(component, property),
-      };
-    });
+    .map(property => ({property, literal: get(component, property)}));
 
   // See ADDITIONAL_PROPERTIES_BY_COMPONENT_TYPE for the reverse check
   switch (component.type) {
     case 'select': {
       const values = component?.data?.values || [];
       finalTranslatableProperties.push(
-        ...values.map(v => {
```

Comment:
I also changed the name from `label` to `values_label` because I found out that there were multiple keys with the same name and the one was overriding the other.